tests/src/ViewModel/SeeMoreLinkTest.php

```diff
@@ -15,21 +15,25 @@ public function it_has_data()
         $data = [
             'name' => 'something',
             'url' => 'http://google.com',
+            'ariaLabel' => 'Related articles',
             'isInline' => true,
         ];
 
-        $link = new SeeMoreLink(new Link($data['name'], $data['url']), $data['isInline']);
+        $link = new SeeMoreLink(new Link($data['name'], $data['url'], $data['ariaLabel']), $data['isInline']);
 
         $this->assertSame($data['name'], $link['name'], 'The names should match');
         $this->assertSame($data['url'], $link['url'], 'The URLs should match');
+        $this->assertSame($data['ariaLabel'], $link['ariaLabel'], 'The ariaLabel should match');
         $this->assertSame($data['isInline'], $link['isInline'], 'The isInline property should be true');
         $this->assertSame($data, array_merge($link->toArray(), ['isInline' => true]));
     }
 
     public function viewModelProvider() : array
     {
         return [
-            [new SeeMoreLink(new Link('something', 'http://google.com'))],
+            'minimal' => [new SeeMoreLink(new Link('something'))],
+            'link only' => [new SeeMoreLink(new Link('something', 'http://google.com'))],
```

Comment:
Have modified minimal as url is optional and introduced a link only

Reply:
The 'url' is required for SeeMoreLink component.